src/neuroconv/datainterfaces/ophys/minian/miniandatainterface.py

```diff
@@ -0,0 +1,76 @@
+from typing import Optional
+
+from pydantic import DirectoryPath
+from pynwb import NWBFile
+
+from ..basesegmentationextractorinterface import BaseSegmentationExtractorInterface
+
+
+class MinianSegmentationInterface(BaseSegmentationExtractorInterface):
+    """Data interface for MinianSegmentationExtractor."""
+
+    display_name = "Minian Segmentation"
+    associated_suffixes = (".zarr",)
+    info = "Interface for Minian segmentation data."
+
+    @classmethod
+    def get_source_schema(cls) -> dict:
+        source_metadata = super().get_source_schema()
+        source_metadata["properties"]["folder_path"]["description"] = "Path to .zarr output."
+        return source_metadata
+
+    def __init__(self, folder_path: DirectoryPath, verbose: bool = False):
```

Comment:
you are missing pydantic validator (check the decorator init in the other interfaces)